source/2019-04-05-the-ember-times-issue-92.md

```diff
@@ -12,10 +12,16 @@ responsive: true
 
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+## [3 Reasons To Volunteer At a Tech Conference 💪🏼](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8)
+
+[@lisaychuang](https://twitter.com/lisaychuang) shares her experience from this year’s EmberConf, and encourages newcomers in any tech community to attend a tech conference (and possibly volunteer!)   
+
+As a volunteer, you can get:
+* A warm welcome to the community
+* Behind the scene access
+* A chill home base during the conference
+
+Curious to learn more? Check out her post on Medium:[3 reasons why you should volunteer at a tech conference](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8).
```

Comment:
The suggestions UI seems to be broken for me rn 😢, but add a space after `Medium:`

Reply:
```suggestion
Curious to learn more? Check out her post on Medium: [3 reasons why you should volunteer at a tech conference](https://medium.com/@lisaychuang/3-reasons-why-you-should-volunteer-at-a-tech-conference-19a0c81d1d8).
```